README.md

```diff
@@ -212,4 +212,4 @@ Thanks for the great jobs of [huxblog Jekyll Theme](https://github.com/Huxpro/hu
 ## Feedback
 If you find any problems, please feel free to [raise an issue](https://github.com/zhaohuabing/hugo-theme-cleanwhite/issues/new) or create a pull request to fix it.
 
-If it's helpful for you, I would appreciate it if you could star this repository, thanks!
\ No newline at end of file
+If it's helpful for you, I would appreciate it if you could star this repository, thanks!

```

Comment:
Not sure what this is, I did the change in the github gui, probably wasn't a good idea.